contracts/credit/CreditFacadeV3.sol

```diff
@@ -589,6 +595,16 @@ contract CreditFacadeV3 is ICreditFacadeV3, ACLNonReentrantTrait {
                     else if (method == ICreditFacadeV3Multicall.setFullCheckParams.selector) {
                         (fullCheckParams.collateralHints, fullCheckParams.minHealthFactor) =
                             abi.decode(mcall.callData[4:], (uint256[], uint16)); // U:[FA-24]
+
+                        if (fullCheckParams.minHealthFactor < PERCENTAGE_FACTOR) {
+                            revert CustomHealthFactorTooLowException(); // U:[FA-24]
+                        }
+
+                        uint256 hintsLen = fullCheckParams.collateralHints.length;
+                        for (uint256 j; j < hintsLen; ++j) {
+                            uint256 mask = fullCheckParams.collateralHints[j];
+                            if (mask == 0 || mask & mask - 1 != 0) revert InvalidCollateralHintException(); // U:[FA-24]
+                        }
```

Comment:
Since full check params are set here, it makes sense to validate them here as well